### PR TITLE
* Build optimisations

### PR DESCRIPTION
**Pull request by Wagnerp**
* Build optimisations

### CODE REVIEW

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Calendar/Classes/CalendarRenderer.cs [hunk 1]
```diff
@@ -1443,37 +1443,40 @@ public virtual void OnDrawTimeScale(CalendarRendererEventArgs e)
...
+                if (unit != null && !unit.Visible)
                 {
                     continue;
                 }
 
-                string hours = unit.Hours.ToString("00");
-                string minutes = unit.Minutes == 0 ? "00" : string.Empty;
-
-                if (!string.IsNullOrWhiteSpace(minutes))
+                if (unit != null)
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Calendar/Classes/CalendarRenderer.cs:1451]: not needed as it is tested directly above.

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Navigator/Classes/OutlookBarButton.cs [hunk 2]
```diff
@@ -93,62 +93,62 @@ public void Dispose()
...
         [DefaultValue(typeof(string), ""), Browsable(true)]
         public string BuddyPage2
         {
-            get => this._BuddyPage2;
-            set => this._BuddyPage2 = value;
+            get => _buddyPage2;
+            set => _buddyPage2 = value;
         }
 
         public string Text
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Navigator/Classes/OutlookBarButton.cs:136]: Should be null able

**Review by Smurf-IV — CHANGES_REQUESTED** [review]

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Calendar/Classes/CalendarRenderer.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Calendar/Classes/CalendarRenderer.cs]: Rolled back

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Navigator/Classes/OutlookBarButton.cs (hunk 2 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Navigator/Classes/OutlookBarButton.cs:136]: Defined as `null`able

**Pull request merged**